mep/pages/templates/pages/snippets/citation.html

```diff
@@ -1,6 +1,22 @@
 <div class="citation">
     <h2>Cite this document</h2>
     <p>
+        <span class="authors">
+        {% spaceless %}
+        {% for person in page.authors %}
+        {% with author=person.value %}
+            {% if forloop.first %}
+            <span>{{ author.last_name }}, {{ author.first_name }}</span>
+            {% elif forloop.last %}
+            <span>, and {{ author.name }}</span>
+            {% else %}
+            <span>, {{ author.name }}</span>
+            {% endif %}
+        {% endwith %}
```

Comment:
Please add a template render test to the page models to check citation construction for no author, one author, two authors, three authors.